selene_sdk/samplers/online_sampler.py

```diff
@@ -179,33 +165,57 @@ def __init__(self,
             if isinstance(validation_holdout, (list,)):
                 self.validation_holdout = [
                     str(c) for c in validation_holdout]
-            else:
+                self._holdout_type = "chromosome"
```

Comment:
I just added this code since I realized we never had it but we probably should have it - it looks like a bug? Basically just specifying `_holdout_type` and doing type-checking if we only specify validation holdout. Let me know if it makes sense to you, I guess we never ever use this configuration so it was not really needed lol

Reply:
yes this makes a lot of sense - good catch!